test/sql/export/export_database.test

```diff
@@ -19,9 +19,16 @@ INSERT INTO integers VALUES (1, 3), (4, 2), (NULL, 1)
 statement ok
 INSERT INTO strings VALUES ('NULL', DATE '1992-01-01'), (NULL, DATE '1993-01-01');
 
+statement ok
+CREATE TABLE "table.with-symbols"(i INTEGER)
+
+statement ok
+INSERT INTO "table.with-symbols" VALUES (1), (4), (NULL)
+
 statement ok
 CREATE VIEW v1 AS SELECT * FROM integers WHERE i>3; --
 CREATE VIEW v2 AS SELECT * FROM integers WHERE i < 3;
+CREATE VIEW "view.with-symbols" AS SELECT * FROM "table.with-symbols" WHERE i < 3;
```

Comment:
Could you also test a table/view that has dots and quotes in it (see e.g. test/sql/catalog/test_quoted_column_name.test)?